services/user-service/index.ts

```diff
@@ -1,22 +1,57 @@
 import express from 'express';
-import { Options } from 'sequelize';
+import { Sequelize, Options } from 'sequelize';
+import session from 'express-session';
+import passport from 'passport';
 import action from "./actions";
 import swaggerJSDoc from 'swagger-jsdoc';
+import { Umzug, SequelizeStorage } from 'umzug';
+import sequelize from './utils/sequelize';
+import passportUtil from './utils/passport';
 
 export default {
-  initialize: (options: { database: Options, swaggerSpecOption?: swaggerJSDoc.Options }) => {
-    if(options.swaggerSpecOption) {
-      const actionPath = __dirname + '/actions/**/*.ts';
-      if(options.swaggerSpecOption.apis) {
-        const apis = options.swaggerSpecOption.apis?.map(e => e);
-        apis?.push(actionPath)
-        options.swaggerSpecOption.apis = apis;
-      } else {
-        options.swaggerSpecOption.apis = [actionPath];
+  initialize: (options?: { swaggerSpecOption?: swaggerJSDoc.Options } | undefined) => {
+    passportUtil.initialize(passport);
+    
+    const umzug = new Umzug({
+      migrations: { 
+        glob: "./services/user-service/migrations/*.ts",
```

Comment:
can we also include the `.js` code in here? since it's looks like the whole migration's things will run automatically on start